frontend/src/components/banner/topBanner.tsx

```diff
@@ -10,11 +10,12 @@ export function TopBanner() {
   const [dangerousInner, setDangerousInner] = useState<null | string | TrustedHTML>(null);
 
   useEffect(() => {
```

Comment:
CC @spwoodcock just wanted to check this with you, this func appears to never be actually called as an IIFE, is it meant to? Just wanted to check, as this bit has some XSS risks if I fixed this and it was actually abandoned etc.

Reply:
Nevermind, seems this bit was off a very very old commit for some reason? Anyway, this is resolved, and fixed the html part 👍